content/docs/reference/pkg/python/pulumi_kubernetes/helm/v2/_index.md

```diff
@@ -259,31 +260,47 @@ Example: A resource created with resource_prefix=”foo” would produce a resou
 
 </dd></dl>
 
+<dl class="class">
+<dt id="pulumi_kubernetes.helm.v2.TextIO">
```

Comment:
I'm going to assume we forgot to regenerate the `pulumi_kubernetes` docs after doing a release, but just to confirm, you have the `pulumi-kubernetes` repo checked out to the latest release branch (looks like `0.25.6`) when running the doc generator?

Reply:
I just rebased and force-pushed. Can you please see if you are looking at the latest?

Reply:
Yes, this is after your rebase.

Reply:
Justin and I spoke offline about this. This diff is fine since the API docs for Kubernetes wasn't generated in the previous docs release.